runtime/src/system_instruction_processor.rs

```
@@ -56,6 +59,16 @@ fn create_system_account(
         );
         Err(SystemError::ResultWithNegativeLamports)?;
     }
+
+    if should_rent_exempt {
+        let rent_calculator = rent::from_keyed_account(&keyed_accounts[RENT_SYSVAR_ACCOUNT_INDEX])
+            .unwrap()
```

you can't unwrap here, you need to throw the error...